engine/src/main/java/com/datatorrent/stram/util/LoggerUtil.java

```diff
@@ -73,6 +79,10 @@ public String apply(@Nullable Level input)
     }
   };
 
+  static {
```

Comment:
The approach to use different threads for master and containers/workers is already implemented in the StramLocalCluster. Running applications in the local mode and in unit tests that use StramLocalCluster should not require any additional explicit calls to the setService().